crates/bevy_sprite/src/texture_atlas.rs

```diff
@@ -10,7 +10,7 @@ use bevy_utils::HashMap;
 /// [Example usage animating sprite.](https://github.com/bevyengine/bevy/blob/latest/examples/2d/sprite_sheet.rs)
 /// [Example usage loading sprite sheet.](https://github.com/bevyengine/bevy/blob/latest/examples/2d/texture_atlas.rs)
 #[derive(Debug, Clone, TypeUuid)]
-#[uuid = "7233c597-ccfa-411f-bd59-9af349432ada"]
+#[uuid = "1629bf14-d49b-4283-8e2c-35cb1d22c59b"]
```

Comment:
as mentioned by Davier, this one should be left unchanged but it should be removed from `Sprite`

Reply:
Done.